### PR TITLE
[daint-mc] Add GROMACS 2020.1 MC

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

**Comment by lucamar** [comment]
Thanks @victorusu: after this pull request, we are going to have the following non-default GROMACS releases in production, on top of the default (older) one:
```
 GROMACS-2019.1-CrayGNU-19.10.eb
 GROMACS-2020-CrayGNU-19.10.eb
 GROMACS-2020.1-CrayGNU-19.10.eb
```
Do you need to keep all of them?

**Comment by victorusu** [comment]
Ciao @lucamar,
We haven't announced any removal of GROMACS versions, so I would like to keep all.
We can announce the removal and keep only the current default and the most up to date one.

**Comment by lucamar** [comment]
Thanks for your reply @victorusu: I suggest to remove from the production list the "intermediate" builds: the modules won't be removed from the system anyway, until you do that manually.

File: jenkins-builds/7.0.UP01-19.10-mc [hunk 1]
```diff
@@ -14,6 +14,7 @@
  GROMACS-2018.6-CrayGNU-19.10.eb                    --set-default-module
  GROMACS-2019.1-CrayGNU-19.10.eb
  GROMACS-2020-CrayGNU-19.10.eb
```
Comment (lucamar) [inline jenkins-builds/7.0.UP01-19.10-mc:16]:
```suggestion
```

**Pull request merged**